subsys/metrics/cdi/src/main/java/org/commonjava/indy/IndyMetricsUtil.java

```diff
@@ -0,0 +1,84 @@
+package org.commonjava.indy;
+
+import com.codahale.metrics.*;
+import com.codahale.metrics.health.HealthCheck;
+import org.commonjava.indy.measure.annotation.IndyMetrics;
+import org.commonjava.indy.measure.annotation.Measure;
+import org.commonjava.indy.measure.annotation.MetricNamed;
+import org.commonjava.indy.metrics.healthcheck.IndyHealthCheckRegistrySet;
+import org.commonjava.indy.metrics.healthcheck.IndyHealthCheck;
+import org.commonjava.indy.metrics.jvm.IndyJVMInstrumentation;
+import org.commonjava.indy.metrics.reporter.ReporterIntializer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.annotation.PostConstruct;
+import javax.annotation.PreDestroy;
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.inject.Any;
+import javax.enterprise.inject.Instance;
+import javax.inject.Inject;
+import java.util.concurrent.TimeUnit;
+
+import static com.codahale.metrics.MetricRegistry.name;
+
+/**
+ * Created by xiabai on 2/27/17.
+ */
+@ApplicationScoped
+public class IndyMetricsUtil {
```

Comment:
Utility classes are typically static. This name was originally good when this only contained static methods, but now it's a component. Can we rename it to IndyMetricsManager or something like that?